compute_endpoint/globus_compute_endpoint/endpoint/endpoint.py

```diff
@@ -109,21 +135,28 @@ def init_endpoint_dir(
                 endpoint_config,
                 config_target_path,
                 multi_tenant,
+                display_name,
             )
         finally:
             os.umask(user_umask)
 
     @staticmethod
     def configure_endpoint(
-        conf_dir: pathlib.Path, endpoint_config: str | None, multi_tenant: bool = False
+        conf_dir: pathlib.Path,
+        endpoint_config: str | None,
+        multi_tenant: bool = False,
+        display_name: str | None = None,
     ):
         ep_name = conf_dir.name
         if conf_dir.exists():
+            print(conf_dir)
             print(f"config dir <{ep_name}> already exists")
```

Comment:
Och; this is a good call.  I wonder how we didn't think to "not hide" this before.  :+1: